controllers/operands/ssp.go

```diff
@@ -256,10 +258,6 @@ func getCommonDicts(list []hcov1beta1.DataImportCronTemplateStatus, crDicts map[
 		}
 
 		if crDict, found := crDicts[dictName]; found {
-			if !isDataImportCronTemplateEnabled(crDict) {
-				continue
-			}
```

Comment:
This would disable user provided DICs with the annotation that don't correspond to an internal DIC. I don't think that's an issue but it is a change in behaviour, as we've seen these would previously still be created.